data_source_examples/README.md

```diff
@@ -4,8 +4,12 @@ If you don't see an example for your data source or are looking for data source
 
 ---
 
-**🔜 Data source specific examples are coming soon.**
+There are currently three data source specific upsert examples available:
+
+- [**PostrgeSQL database**](postgresql/using_airtable.js/) - import rows from multiple PostgreSQL tables into an Airtable base while maintaining relationships
```

Comment:
Could potentially reword this to :
import rows from multiple PostgreSQL tables into an Airtable base while maintaining **table to table** relationships